back/app/models/embeddings_similarity.rb

```diff
@@ -0,0 +1,25 @@
+# == Schema Information
+#
+# Table name: embeddings_similarities
+#
+#  id                  :uuid             not null, primary key
+#  embedding           :vector(1024)     not null
+#  embeddable_type     :string           not null
+#  embeddable_id       :uuid             not null
+#  embedded_attributes :string
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#
+# Indexes
+#
+#  index_embeddings_similarities_on_embeddable           (embeddable_type,embeddable_id)
+#  index_embeddings_similarities_on_embedded_attributes  (embedded_attributes)
+#  index_embeddings_similarities_on_embedding            (embedding) USING hnsw
+#
+class EmbeddingsSimilarity < ApplicationRecord
+  belongs_to :embeddable, polymorphic: true
+  has_neighbors :embedding
```

Comment:
Not sure if we still need this, since we're no longer using their `nearest_neighbors` method. The gem could then also be removed. Alternatively, we submit a PR to support a threshold for the `nearest_neighbors` method, but this is more effort and we ran out of time.